ml.py

```diff
@@ -331,6 +357,11 @@ def fit(self, X, y, **kwargs):
             self.cv_results_['reg'].append(reg)
             self.cv_results_['metrics'].append(metrics)
         self._convert_report_to_df()
+        # TODO: Reverse back if add_box_cox_target is True
+        if self.add_box_cox_target:
+            self.oof_train_ = inv_boxcox1p(self.oof_train_, self.lmda)
```

Comment:
This line can be removed. self.oof_train_ comes from y_pred which is already reversed from _get_reg_metrics